### PR TITLE
Doc polish

### PR DESCRIPTION
**Pull request by pgicking**
Is this what you had in mind @joshproehl? Theres not much to say, we've only got 5 general codes.

### CODE REVIEW

File: behat.yml [hunk 1]
```diff
@@ -2,4 +2,4 @@
 default:
     context:
         parameters:
-            base_url: http://localhost/api-test/index.php/
+            base_url: http://localhost:8080/api/index.php/
```
Comment (joshproehl) [inline behat.yml]: @pgicking: Why this change?

File: testing_database_setup_script [hunk 2]
```diff
@@ -32,4 +32,4 @@ fi
 
 # if you remove the comment from the line below, the testing for the api will
 # run auto-magically after the testing database has been set up.
-# vendor/bin/behat
+ vendor/bin/behat
```
Comment (joshproehl) [inline testing_database_setup_script]: @pgicking: Is this a default we want to be enabled? (i.e. did you intend to commit this?)

File: public/TrafficStats.php [hunk 3]
```diff
@@ -33,8 +33,18 @@ class TrafficStats{
...
+     * <br />
+     * Possible response codes:<br />
+     * 200 - No errors<br />
+     *<br />
+     * 400 - Couldn't get stations from users given taps, user somehow tapped more than 200 meters away from a station<br />
+     * 400 - Given stations refer to different highways, please choose only one highway<br />
+     *<br />
+     * 404 - Couldn't find the requested object, usually because the given ID doesnt exist in the database<br />
+     *<br />
+     * 412 - The sent JSON object is empty<br />
```
Comment (joshproehl) [inline public/TrafficStats.php]: :+1: All of this looks great! :-)

File: behat.yml (hunk 1 above)
Comment (pgicking) [inline behat.yml]: wasn't meant to be committed

File: testing_database_setup_script (hunk 2 above)
Comment (pgicking) [inline testing_database_setup_script]: also wasn't meant to be committed, my bad. anyway to specifically exclude these two files?

**Comment by pgicking** [comment]
fixed the accidental commits

**Comment by severity** [comment]
@joshproehl: is this ready to be merged? :)

**Comment by joshproehl** [comment]
FYI @pgicking when you're doing a PR like this it's fair game to force-push an update, i.e. simply remove the offending changes and force push. No harm in having "revert" commits in general, but sometimes it's nice to keep the history clean. :smile: 

Closing, will be merged in at 78cdf7b35903cc0abfb9ca2e42969c56bce0f445
